### PR TITLE
Release 2.2.2

### LINKED ISSUE #361: New (minor) release
Can there be a new (minor) release such that conda-forge package gets the new H5Easy features? Maybe after merging https://github.com/BlueBrain/HighFive/pull/343

### PR DESCRIPTION
**Pull request by tdegeus**
Fixes #361

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+## Version 2.2.2 - 2020/07/30
```
Comment (matz-e) [inline CHANGELOG.md:1]: FWIW, we could format dates according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)…

**Comment by alkino** [comment]
It should be merged in 2.2.1 and not in master ?

File: CHANGELOG.md (hunk 1 above)
Comment (tdegeus) [reply CHANGELOG.md:1]: So `YYYY-MM-DD`?

**Comment by tdegeus** [comment]
@alkino Looking at the rest of the repo it should be the master no? I would find that most logical also...

**Comment by alkino** [comment]
I made the 2.2.1 as a fix branch. If you go from 2.2 to 2.2.1 you got only fixes.

But you can do as you want.

**Pull request merged**